src/components/language_picker.html

```diff
@@ -0,0 +1,39 @@
+{% import "Macros" as macros %}
+<button id="language-toggle{{ counter }}" class="icon-button" type="button"
+    title="Change language" aria-label="Change language"
+    aria-haspopup="true" aria-expanded="false"
+    aria-controls="language-list{{ counter }}" style="{{ attributes | get(key="style", default="")  }}">
```

Comment:
Reading attributes passed in HTML

Reply:
This should go away in a MVP — I would like to keep this super simple at first.